docs/main/prerequesites.md

```diff
@@ -11,10 +11,10 @@ Any hardware that runs one of the supported operating systems will do!
 
 The following operating systems are **officially** supported:
 
-| Distribution | Release          |  Architecture       |
+| Distribution | Release          | Architecture        |
 | ------------ | ---------------- | ------------------- |
 | Raspbian     | Jessie / Stretch | ARM                 |
-| Ubuntu       | 16.04 / 16.10    | ARM / x86_64        |
+| Ubuntu       | 18.04.1 / 18.10  | ARM / x86_64        |
```

Comment:
16.04 and 16.10 should still be supported, as they map to Debian Stretch.

Reply:
updated to state `16.x` & `18.x` support.